qhub/template/{{ cookiecutter.repo_directory }}/infrastructure/state.tf

```diff
@@ -26,4 +26,13 @@ terraform {
     skip_metadata_api_check     = true
   }
 }
+{% elif cookiecutter.provider == "azure" -%}
+terraform {
+  backend "azurerm" {
+    resource_group_name  = "{{ cookiecutter.project_name }}-terraform-state"
+    storage_account_name = "{{ cookiecutter.project_name }}storage"
```

Comment:
I may need to add some random nubmer to this storage account since I think it needs to be globally unique